packages/remark-shiki-twoslash/src/index.ts

```diff
@@ -95,7 +93,7 @@ const amendSettingsForDefaults = (settings: UserConfigSettings) => {
 
 const parsingNewFile = () => includes.clear()
 
-////////////////// The Remark API
```

Comment:
The old comment was triggering some weird highlighting in my editor so I updated it even though it was unnecessary.